### ISSUE TITLE
Testing Data

### ISSUE THREAD

**Issue by Prinstin** [jiny2001/dcscn-super-resolution#35]
Does the training data and testing data image type need to be consistent?
If I use my own training data and testing data,and place them in the TrainData and TestData directory respectively.For training,can I use --dataset=TrainData --test_dataset=TestData ? Or modify the test_dataset definition in arg.py ?

**Comment by rafaelleinio** [comment]
I think he uses the set5 in the training for testing. So when I was training I just replaced the images in the folder set5 with my test images.

**Comment by jiny2001** [comment]
Yep, using --dataset=TrainData --test_dataset=TestData is the best way. Please note you should put those directory under "data" directory.

set5 is the default of "testing" dataset. The default "training" dataset is BSD200 now. However, I don't recommend you to replace those images in the directory. Because when you use do_bechmark=True, set5 will be used as a benchmarking test set.

Thank you for using my code. I will add readme the detail of how to train with your own dataset in some days. :)

**Comment by Prinstin** [comment]
Thank you very much.

Does the "concatenate" in your model structure mean that all CNN layers' features are weighted averaged during training and evaluating?

**Comment by Prinstin** [comment]
or averaged only?

**Comment by jiny2001** [comment]
No, it is not averaging. In here "concatenate" means just adding the features. For ex, when CNN01 outputs 32 features and CNN02 outputs 16 features, the concatenated output has 48 features.

**Comment by Prinstin** [comment]
How about the '+' before final output? The output fo last CNN is summed with the corresponding pixel of the bicubic upsampling image ?

**Comment by jiny2001** [comment]
You mean, adding bi-cubic upsampling data? In this case, '+' means just adding the value for each corresponding pixels. You're explanation is correct. This is called residual block. :)

**Comment by Prinstin** [comment]
OK.Thank you for your help in your busy schedule.
